complete/style.css

```diff
@@ -0,0 +1,155 @@
+/* === HEADER STYLE === */
+
+.header {
+  background: url(images/header-bg.png) no-repeat center;
+  background-size: cover;
+  width: 100wv;
+}
+  
+/* === LOGO STYLE === */
+
+.top-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.5)
+}
+
+.logo {
+  padding: 14px 16px;
+  flex: 1;
+}
+/* === TOP NAVBAR STYLE === */
+
+.top-navbar {
+  flex:2;

```

Comment:
![header-menu](https://user-images.githubusercontent.com/2418147/42778501-8f4aef4e-893d-11e8-8784-1ba561c825a5.jpg)

When resizing the browser, the behavior of the menu is not as how we pictured it.
Here, in this image, it looks like the button `contact us` would have room to fit in there, yet it shows on a second line?

Here is our ideal behavior of the header when resizing the browser smaller:
- the menu is *always* on one line
- when there is no more room, remove the social icons completely
- if even afterwards there is no more room, then stack everything vertically (with the logo AT the top)
- when the menu is vertical, it should have 100% width

HINT: The current issue is due to using flex: 1 for logo and social icons and flex 2 for the menu.
We would recommend removing them and only add flex 1 to the menu

Reply:
Here are some more exampls of an ideal behavior

![menu2](https://user-images.githubusercontent.com/2418147/42779507-f4d39e80-8940-11e8-8088-c89e4f2dc7ef.jpg)


![menu3](https://user-images.githubusercontent.com/2418147/42779512-f78f903e-8940-11e8-8628-932343a81843.jpg)